src/components/Basic/Switch/Switch.module.css

```
@@ -0,0 +1,61 @@
+.Switch {
+    display: flex;
+    align-items: center;
+    cursor: pointer;
+}
+
+.Control {
+    width: 3.5vmax;
+    height: 2vmax;
+    position: relative;
+    border-radius: 1vmax;
+    border: 0.1vmax solid #00DA9A;
+    background-color: #D9D9D9;
+}
+
+.LabelLeftEnabled {
+    margin-right: 0.3vmax;
+}
+
+.LabelLeftEnabled {
+    margin-left: 0.3vmax;
+}
+
+.Disabled {
+    opacity: 0.8;
+    pointer-events: none;
+}
+
+.Indicator {
+    position: absolute;
+    width: 100%;
+    height: 100%;
+    border-radius: 0.9vmax;
+    top: 0;
+    left: 0;
+    z-index: 1;
+    background-color: #00DA9A;
+    opacity: 0;
+    transition: opacity ease-in-out 0.2s;
+}
+
+.IndicatorChecked {
+    opacity: 1;
+}
+
+.Handle {
+    position: absolute;
+    width: 1.4vmax;
+    height: 1.4vmax;
+    border-radius: 50%;
+    top: 0.2vmax;
+    z-index: 2;
+    left: 0.2vmax;
+    background-color: #FFFFFF;
+    transition: transform ease-in-out 0.2s, background-color ease-in-out 0.2s;
+}
+
+.HandleChecked {
+    background-color: #006e4d;
+    transform: translateX(1.5vmax);
```

```suggestion
    transform: translateX(100%);
```
If for some reason users change the width of the control the handle won't display properly

---

Fixed. Thanks!